### PR TITLE
Allow importing of multiple symbols in a single import statement

### LINKED ISSUE #808: Import multiple symbols in a single from-import statement
## Abstract
From-import directive should support multiple import symbols. This can be achieved as a syntactic sugar.

To avoid cases like the following:
```zokrates
from "./module.zok" import func1
from "./module.zok" import func2
from "./module.zok" import func3
...
```
one should be able to import all needed functions in a single line:
```zokrates
from "./module.zok" import func1, func2, func3
```

## Motivation
This is a common feature in most languages, as it is easy to implement, we should support this as well.

## Specification
The implementation should cover all of the following cases:

```zokrates
# without aliases
from "./source.zok" import func1, func2

# with aliasing
from "./source.zok" import func1 as f1, func2
from "./source.zok" import func1 as f1, func2 as f2
```

## Backwards Compatibility
No breaking changes.

### PR DESCRIPTION
**Pull request by dark64**
Closes https://github.com/Zokrates/ZoKrates/issues/808

### CODE REVIEW

File: changelogs/unreleased/809-dark64 [hunk 1]
```diff
@@ -0,0 +1 @@
+Add ability to import multiple symbols in a single import statement
```
Comment (Schaeff) [inline changelogs/unreleased/809-dark64]:
```suggestion
Add the ability to import multiple symbols in a single import statement
```

File: zokrates_book/src/language/imports.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ from "./path/to/my/module" import MySymbol
 // `MySymbol` is now in scope.
 ```
 
+To import multiple modules with a single import statement, separate the module names with commas:
```
Comment (Schaeff) [inline zokrates_book/src/language/imports.md]:
```suggestion
To import multiple symbols with a single import statement, separate the symbol names with commas:
```

File: zokrates_core/src/imports.rs [hunk 3]
```diff
@@ -56,6 +56,25 @@ impl From<io::Error> for Error {
     }
 }
 
+#[derive(PartialEq, Clone)]
+pub enum ImportKind<'ast> {
```
Comment (Schaeff) [inline zokrates_core/src/imports.rs]: Could this be simplified to only the Multiple variant or removed completely?

**Review by Schaeff — CHANGES_REQUESTED** [review]

File: zokrates_book/src/language/imports.md (hunk 2 above)
Comment (dark64) [reply zokrates_book/src/language/imports.md]: Oh, how did I not notice this :facepalm: Nice catch!

**Review by Schaeff — APPROVED** [review]

**Pull request merged**